### ISSUE TITLE
Issue with English (Canada) language pack 4.1.0.1

### ISSUE THREAD

**Issue by fballem** [joomla/core-translations#197]
### Steps to reproduce the issue
1. Install English (Canada) language pack.
2. Enable as a content language.
3. Create a category and set language to English (Canada)
4. Save.
5. Jooomla error screen appears.

### Expected result
should be able to save the Category.

### Actual result
Joomla Error screen appears.

### System information (as much as possible)
see additional comments

### Additional comments
/** * en-CA localise class. * * @since  1.6 */abstract class En_GBLocalise{    /**     * Returns the potential suffixes for a specific number of items     *     * @param   integer  $count  The number of items.

**Comment by zero-24** [comment]
I have moved this to the core-translations repo cc @tecpromotion

**Comment by tecpromotion** [comment]
I tested this locally under Joomla! 4.0.1 and was able to save the category without errors.
Tested locally with PHP 7.4.27 and 8.0.15
10.6.4-MariaDB

localise.php looks fine to me in this package https://downloads.joomla.org/language-packs/translations-joomla4/downloads/joomla4-english-canada/4-1-0-1

@fballem can you give more information about your system?
Which MySQL do you use?
Which PHP version do you use?
Can you also recreate the error with Danish?

@particthistle can you please check this too. Thanks.

**Comment by infograf768** [comment]
**The localise.php for en-CA is fine for admin but NOT for site.**
Priority is always for the site file when present.
Code:
```
defined('_JEXEC') or die;

/**
 * en-CA localise class.
 *
 * @since  1.6
 */
abstract class En_GBLocalise
{
	/**
	 * Returns the potential suffixes for a specific number of items
	 *
	 * @param   integer  $count  The number of items.
	 *
	 * @return  array  An array of potential suffixes.
	 *
	 * @since   1.6
	 */
	public static function getPluralSuffixes($count)
	{
		if ($count == 0)
		{
			return array('0');
		}
		elseif ($count == 1)
		{
			return array('ONE', '1');
		}
		else
		{
			return array('OTHER', 'MORE');
		}
	}
}
```

**Comment by tecpromotion** [comment]
> **The localise.php for en-CA is fine for admin but NOT for site.** Priority is always for the site file when present.

Many thanks @infograf768 

@partichistle please check and fix in crowdin. Thanks.

**Comment by fballem** [comment]
A question, if I may. Is the fix to change En_GBLocalise to En_CALocalise?

**Comment by tecpromotion** [comment]
> A question, if I may. Is the fix to change En_GBLocalise to En_CALocalise?

Yes. This should fix the issue. I approved this already in crowdin.

**Comment by tecpromotion** [comment]
see #203
